tests/spread/ubuntu-24.04/test-cmd/spread.yaml

```diff
@@ -0,0 +1,21 @@
+project: test-cmd
+
+backends:
+  craft:
+    type: adhoc
```

Comment:
Is this test working? Shouldn't we be testing with a type: craft backend?

Reply:
Changed :-)